changelogs/fragments/1765-proxmox-params.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - proxmox - refactored some parameter validation code into use of ``env_fallback``, ``required_if``, ``required_together``, ``required_one_of`` (https://github.com/ansible-collections/community.general/pull/1765).
```

To make sure people know this applies not just to the module called `proxmox`, but also some of the others:
```suggestion
  - proxmox* modules - refactored some parameter validation code into use of ``env_fallback``, ``required_if``, ``required_together``, ``required_one_of`` (https://github.com/ansible-collections/community.general/pull/1765).
```